htdocs/ticket/class/ticket.class.php

```diff
@@ -1242,7 +1242,8 @@ public function loadCacheTypesTickets()
 
 		$sql = "SELECT rowid, code, label, use_default, pos, description";
 		$sql .= " FROM ".MAIN_DB_PREFIX."c_ticket_type";
-		$sql .= " WHERE active > 0";
+		$sql .= " WHERE entity = ".getEntity('c_ticket_type');
```

Comment:
With getEntity in a SQL, you must use
...entity IN (".getEntity().")"

Reply:
@eldy not for dictionary table, getEntity return 1 if all entities use the same dictionary or the current entity if the dictionary is personalized

Reply:
So even in this case, we can use the IN.
For a more uniform situation, it should be better to have IN everywhere for getEntity, even if in execution th in become a IN (1)

Reply:
@eldy ok it's fixed